attributes/network/11_ledger_tokens/cddl/1_visual_logo.cddl

```diff
@@ -0,0 +1,45 @@
+; A single logo that can be shown for representing the token. A token can
+; have a set of logos that can represent visually its identity.
+single-visual-token-logo = single-visual-token-logo-unicode
+                         / single-visual-token-logo-image
+
+; A unicode logo.
+; For Bitcoin this would be U+20BF ("₿").
+single-visual-token-logo-unicode = {
+    ; Type.
+    0 => 0,
+
+    ; Unicode code point. This isn't limited to a single glyph but can be
```

Comment:
I would rephrase to make it clear that we don't actually want the Unicode code point here.